src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -65,14 +64,20 @@ const styles: Styles<Theme, object> = {
             borderRadius: '50%',
         },
     },
+    clickableLocation: {
+        cursor: 'pointer',
+        color: 'blue',
+    },
+
+    clickableLocationDark: {
+        cursor: 'pointer',
+        color: '#1cbeff',
+    },
 };
 
-const genMapLink = (location: string) => {
-    try {
-        const location_id = locations[location.split(' ')[0] as keyof typeof locations];
-        return 'https://map.uci.edu/?id=463#!m/' + location_id;
-    } catch (err) {
-        return 'https://map.uci.edu/';
+const selectBuilding = (buildingFocusInfo: BuildingFocusInfo) => {
+    if (buildingFocusInfo.location !== 'TBA') {
```

Comment:
This breaks if you click on a class where the location is "ONLINE" (try it with Art 1C)